Jenkinsfile

```diff
@@ -0,0 +1,51 @@
+pipeline {
+  agent {
+    kubernetes {
+      yamlFile 'kubes-pod.yaml'
+      defaultContainer 'ubuntu'
+      activeDeadlineSeconds 3600
+      idleMinutes 15
+    }
+  }
+  stages {
+    stage('Init') {
+      steps {
+        checkout scm
+      }
+    }
+    stage('Setup build') {
+      when { not { buildingTag() } }
+      steps {
+        script {
+          target = "install"
+        }
+      }
+    }
+    stage('Setup release') {
+      when { buildingTag(); not { changeRequest() }  }
+      steps {
+        script {
+          target = "install deploy -P release  --settings settings.xml"
```

Comment:
Shouldn't this have `-P rpm` as well?

Reply:
There are some other useful flags we've used in the past as well:

* `-U` - Update all package versions (avoids negative caching)
* `-B` - Batch mode short circuits any prompts for user input
* `-Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn` - Reduces logging

Reply:
Ref:
https://github.com/ArpNetworking/metrics-aggregator-daemon/blob/master/.github/workflows/build.yml#L32

Reply:
Updated. What is the stuff from today about the pooled connections and keep-alives?

Reply:
It wasn't a problem on Travis, and I would be surprised it was a problem egressing your build network. I believe you do pay a penalty wrt performance with the settings that were prescribed for Azure. I'm okay going back to baseline here.